dotnet/samples/TwentyQuestions/appsettings.Development.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "AllowedHosts": "*",
+  "BOT_ID": "$botId$",
+  "BOT_PASSWORD": "$bot-password$"
```

Comment:
Add `OPENAI_API_KEY`?

Reply:
Ah yes, I'm considering to use something like:
``` json
"OpenAI": {
    "ApiKey": "..."
},
"AzureOpenAI": {
    "ApiKey": "",
    "Endpoint": "",
    ...
}
```

To separate OpenAI and AzureOpenAI